### ISSUE TITLE
Hello?

### ISSUE THREAD

**Issue by Nickduino** [chrisdeely/echo-harmony-bridge#1]
Hey!

Are you still developing this?

**Comment by chrisdeely** [comment]
@Nickduino No, this is not under active development. [Logitech has released native support for Amazon Echo devices](https://support.myharmony.com/en-us/harmony-experience-with-amazon-alexa) which makes this somewhat unnecessary. Is there a specific issue you are experiencing?

**Comment by Nickduino** [comment]
@chrisdeely Thanks for the answer, I have to admit I didn't have high hopes...

The main issue with the official Logitech skill is that it's only available in English and German (which is a pitty for a swiss company since the country has four official languages)...


But I wasn't interested in the Alexa functionality, I contacted you because you mentionned fauxmo and pyHarmony: I want to control a fauxmo device from the Harmony hum. The idea is to [manage my Somfy shutters](https://github.com/Nickduino/Pi-Somfy) from the remote
